molecule/_shared/base.yml

```diff
@@ -30,10 +30,10 @@ provisioner:
   playbooks:
     prepare: ../_shared/prepare.yml
     converge: ../_shared/converge.yml
+    verify: ../_shared/verify.yml
```

Comment:
Technically, this commit should also contain the addition of the new `verify.yml` file. However, it's added in a later commit so I won't nitpick about that.